src/classes/traits/proteus.php

```diff
@@ -88,24 +92,24 @@ public function proteus_welcome_panel() {
 							<p><?php echo sprintf( 'Go to <span class="sandbox-setting-path">Appearance > <a href="%1$s">Menus</a></span>. Select the <em>Main Menu</em> from the dropdown at the top and click on the <em>Select</em> button. You will be able to add or remove menu items to fit your wishes. Don\'t forget to click on the <em>Save Menu</em> button. <br>(if you remove a page from the menu, you can still find it in <em>Pages > All Pages</em>)', admin_url( 'nav-menus.php' ) ); ?></p>
 						</div>
 					</div>
 
 				  <div class="welcome-panel-column">
 						<div class="welcome-panel-inner">
 							<!-- Edit content -->
-							<h3><?php esc_html_e( 'Edit website content', 'woocart-defaults' ); ?></h3>
+							<h3>5. <?php esc_html_e( 'Edit website content', 'woocart-defaults' ); ?></h3>
 							<p><?php echo sprintf( 'Go to <span class="sandbox-setting-path">Pages > <a href="%1$s">All Pages</a></span> and edit one of the pages (example: Home page). To edit text and images, hover the mouse over widgets and select <em>edit</em>. Don\'t forget to click on the <em>Update</em> button to save your changes. You can also copy & paste widgets by right clicking on them (works from page to page as well). To make it even easier, select the <em>Live Editor</em> in the small menu above the page.', admin_url( 'edit.php?post_type=page' ) ); ?></p>
 						</div>
 					</div>
 
 					<div class="welcome-panel-column welcome-panel-last">
 						<div class="welcome-panel-inner">
 							<!-- Export content -->
-							<h3><?php esc_html_e( 'Export your website content', 'woocart-defaults' ); ?></h3>
-							<p><?php echo sprintf( 'Go to <span class="sandbox-setting-path">Tools > <a href="%1$s">Export Content</a></span>. There you can export this website content with all the changes you made, so that you will be able to import it on your own WordPress site (on your domain). All you have to do is to <a href="%2$s" target="_blank">purchase the %3$s theme</a> and follow the steps on the <a href="%1$s">Export content</a> page.', admin_url( 'tools.php?page=pt-sandbox-ocde' ), $this->purchase_link( 'wp-steps' ), explode( ' ', ( wp_get_theme() )->get( 'Name' ), 2 )[0] ); ?></p>
+							<h3>6. <?php esc_html_e( 'Export your website content', 'woocart-defaults' ); ?></h3>
+							<p><?php echo sprintf( 'Go to <span class="sandbox-setting-path">Tools > <a href="%1$s">Export Content</a></span>. There you can export this website content with all the changes you made, so that you will be able to import it on your own WordPress site (on your domain). All you have to do is to <a href="%2$s" target="_blank">purchase the %3$s theme</a> and follow the steps on the <a href="%1$s">Export content</a> page. Or just continue hosting with <a href="" target="_blank">WooCart!</a>', admin_url( 'tools.php?page=pt-sandbox-ocde' ), $this->purchase_link( 'wp-steps' ), explode( ' ', ( wp_get_theme() )->get( 'Name' ), 2 )[0], 'https://woocart.com/paddle/proteus?store_id=' . $_SERVER['STORE_ID'] ); ?></p>
```

Comment:
Add the same link as on the CTA button.

Reply:
Or is this already? Don't understand enough code. I just saw blank ahref.

Reply:
The link was there but the reference to it was missing in the link. Good catch 👍